### DISCUSSION TITLE
Deny anonymous connections

### DISCUSSION THREAD

**Discussion by NyCodeGHG** [FerretDB/FerretDB#4288] [General]
Hi, I couldn't find a way to only allow authenticated connections to FerretDB?
Is there a way to do this and I just couldn't find it?

**Comment by AlekSi** [comment]
You need to enable authentication in PostgreSQL. It is documented there: https://docs.ferretdb.io/security/authentication/

**Reply by NyCodeGHG** [reply]
I have authentication enabled in postgresql, but when I don't specify a username in the ferretdb postgres uri i get the same error as in #4065

**Reply by chilagrow** [reply]
Hi @NyCodeGHG what is the error you get? https://github.com/FerretDB/FerretDB/issues/4065 specifies two different errors. Could you share more details including the driver and its version, also PostgreSQL version please?
